gulpfile.js/build.js

```diff
@@ -17,6 +17,7 @@
 'use strict';
```

Comment:
This file did get quite large, is there a reasonable way to split it up?

Reply:
sure

Reply:
that'd be great to do before we get this in